### DISCUSSION TITLE
Github not responding to my ticket

### DISCUSSION THREAD

**Discussion by clslvs** [community/community#142973] [Other Feature Feedback, Questions, & Ideas]
It's almost 3 weeks and i haven't still got any response towards my ticket and i need my account to be restored because it is locked by 2FA and i can't access it no more

**Comment by aymena80s545** [comment]
https://t.me/Helloo_FRIENDD

**Comment by ebndev** [comment]
Hi @clslvs,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
